### DISCUSSION TITLE
Release-5.2.0 Post install script (user assign role issue)

### DISCUSSION THREAD

**Discussion by gohilamariappan** [Sunbird-Ed/Community#553] [Bugs]
Hi Team!
@santhosh-tg 
We are getting the below error while assigning the roles to the user. 

##############
TASK [post-install : Assigning roles to user] **********************************
fatal: [localhost]: FAILED! => {
    "access_control_allow_origin": "*", 
    "changed": false, 
    "connection": "close", 
    "content": "You do not have permission to perform this operation", 
    "content_length": "52", 
    "content_type": "text/plain; charset=UTF-8", 
    "date": "Mon, 19 Jun 2023 10:57:36 GMT", 
    "elapsed": 0, 
    "redirected": false, 
    "status": 403, 
    "url": "https://compass-dev.tarento.com/api/user/v2/role/assign", 
    "via": "kong/0.14.1", 
    "x_kong_proxy_latency": "12", 
    "x_kong_upstream_latency": "4", 
    "x_ratelimit_limit_hour": "5000", 
    "x_ratelimit_remaining_hour": "4999", 
    "x_request_allowed": "no"
}

MSG:

Status code was 403 and not [200]: HTTP Error 403: Forbidden
##########

Regards,
Gohila

**Comment by tsprasath** [comment]
@gohilamariappan  Please use the postman collection instead of postscript jobs  https://www.postman.com/sunbird-building-blocks/workspace/sunbird-ed/collection/25186239-6547f9fc-5f16-495c-b5e0-d76f5a78d66d

**Comment by gohilamariappan** [comment]
@tsprasath Thank you. Used postman collection for user creation.

Regards,
Gohila
